CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 ### Mixin
 * [BUGIX] Dashboards: Fix legend showing `persistentvolumeclaim` when using `deployment_type=baremetal` for `Disk space utilization` panels. #3173
+* [ENHANCEMENT] Dashboards: Add optional row about the Distributor's metric forwarding feature to the `Mimir / Writes` dashboard. #3182
```

Comment:
[nit] Do you mind moving it above the bugfix one (which also has a typo :))

Reply:
sure, fixed both: da2740fe8b